engine/src/main/java/org/terasology/world/block/tiles/WorldAtlasImpl.java

```diff
@@ -205,7 +224,7 @@ private void buildAtlas() {
 
         int numMipMaps = getNumMipmaps();
         ByteBuffer[] data = createAtlasMipmaps(numMipMaps, TRANSPARENT_COLOR, tiles, "tiles.png");
-        ByteBuffer[] dataNormal = createAtlasMipmaps(numMipMaps, UNIT_Z_COLOR, tilesNormal, "tilesNormal.png");
+        ByteBuffer[] dataNormal = createAtlasMipmaps(numMipMaps, UNIT_Z_COLOR, tilesNormal, "tilesNormal.png", tilesGloss);
```

Comment:
Hmmm. So here we are tying normals with glossiness... I wonder what this imply if one day we want to use normals for environmental reflections instead of simple glossyness. Just thinking out loud though, for the time being it's ok.